### PR TITLE
Add -p HOST_MAX argument.

### PR DESCRIPTION
**Pull request by darko-poljak**
When operating hosts in parallel do it in HOST_MAX maximum parallel host processes.
If argument is not specified, by default CPU count is used.

### CODE REVIEW

**Comment by dheule** [comment]
Cool, we need this feature, since -p with 200 hosts (all complete parallel) was to heavy for the central system and now we can limit that. 

Looking forward to see this in the next release ;-)

**Comment by telmich** [comment]
@dheule we are having similar problems here now -- great to hear that you are also in a nice bigger area of hosts to configure!

File: docs/src/man1/cdist.rst [hunk 1]
```diff
@@ -153,9 +153,10 @@ Configure/install one or more hosts.
 
 .. option:: -j [JOBS], --jobs [JOBS]
 
-    Specify the maximum number of parallel jobs. Global
-    explorers, object prepare and object run are supported
-    (currently in beta).
+    Operate in parallel in specified maximum number of
+    jobs. Global explorers, object prepare and object run
+    are supported. Without argument CPU count is used by
+    default. Currently in beta.
```
Comment (darko-poljak) [inline docs/src/man1/cdist.rst]: This is for -j option which specifies maximum jobs for global explorers, object prepare and object run within one host.

-p is not in beta.

So we now have -p HOST_MAX and for each host process -j JOBS parallel jobs.

**Comment by telmich** [comment]
Perfect, sorry for the confusion!

**Pull request merged**